addons/l10n_cz/tests/test_moves.py

```diff
@@ -1,5 +1,7 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+from freezegun import freeze_time
```

Comment:
You can remove the freeze_time, it's useless
All lines with date `2024-12-31` would be a different day than today.